Docker/Dockerfile-Newest

```
@@ -41,7 +41,7 @@ RUN rm -f /etc/apache2/conf.d/languages.conf /etc/apache2/conf.d/info.conf \
 	# https://bugs.php.net/bug.php?id=78681 \
 	sed -r -i "s/php8_module/php_module/" \
 		/etc/apache2/conf.d/php8-module.conf && \
-	ln -s /usr/bin/php8 /usr/bin/php && \
+	ln -s /usr/bin/php8 /usr/bin/php ; \
```

I'm not sure I follow? If it fails we assume all's well and we want to continue with the rest?

---

Yes indeed. I do expect that php8 will be symlinked from php out of the box at some point

---

And `ln` fails if the target already exists

---

Seems (mildly) risky to depend on failure to be intentional in that sense. :-)

---

It is "only" an image for development, using an unstable base. `php` is not installed if not for what we have ourselves installed higher up, so it cannot be something else. So it is just a way to say *Create symlink if it does not exist already* . Feel free to propose a nicer syntax if you like :-)

---

Hence mildly.

I don't know if this is a nicer syntax, but this would be the correct behavior:
```suggestion
	if [ ! -f /usr/bin/php ]; then echo ln -s /usr/bin/php8 /usr/bin/php; else true; fi && \
```

---

Let's go for it :-)